workers/unity/Packages/com.improbable.gdk.tools/DevAuthTokenUtils.cs

```diff
@@ -9,16 +9,26 @@ namespace Improbable.Gdk.Tools
 {
     public static class DevAuthTokenUtils
     {
+        public static string DevAuthToken => PlayerPrefs.GetString(DevAuthTokenKey);
```

Comment:
do we want to store them in the PlayerPrefs or the EditorPrefs (I genuinely don't know whether that actually amkes a difference)

Reply:
EditorPrefs are for all projects with a specific version of the Unity editor - so if a user generated a DAT in project X on 2019.1, the DAT would exist for them in project Z on 2019.1

PlayerPrefs are tied to the company/product name of the project - so different projects would have different DATs (assuming the product names in the project are different)